simpleimage/make_rootfs.sh

```diff
@@ -144,6 +170,10 @@ exit 101
 EOF
 chmod a+x "$DEST/usr/sbin/policy-rc.d"
 
+# Mount dev
+mount --bind /dev $DEST/dev
+mount --bind /dev/pts $DEST/dev/pts
```

Comment:
These mounts make no sense - they are only needed in the chroot (if at all) and thus should be added to the do_chroot function.